modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -0,0 +1,144 @@
+= Export Anypoint Platform Data to Third-Party Apps
+
+The Telemetry Exporter feature enables you to export xref:access-management::audit-logging.adoc[_audit logs_] to third-party observability and analytics apps, such as Azure Monitor, Datadog, and Splunk.
+//trace data should be a link once the topic exists
+
+To use the Telemetry Exporter feature, you must create a _connection_ with a third-party app to authorize Anypoint Platform to share data. You must then create a _configuration_ to specify the frequency of data transfers. You can also edit and delete connections and configurations.
+
+// add back to configs: "configuration to specify the types of data Anypoint Platform exports as well as..."
+
+
+== Before You Begin
+
+Before you configure the Telemetry Exporter feature, ensure that you have:
+
+* An Anypoint Platform account with the Telemetry Exporter Administrator and Telemetry Exporter Configuration Manager permissions
```

Comment:
We have to specify somewhere that you need Telemetry Exporter Administrator permission to create or modify Connections.

And, you need Telemetry Exporter Configuration Manager permission to create or modify Configurations.